### PR TITLE
[WIP] Add Spec Files + PreCommit Test

### PR DESCRIPTION
**Pull request by svetakvsundhar**
This PR adds a `.spec` file for one notebook test, as well as the PreCommit test to trigger this.

After this PR is in, `.spec` files will be added to all existing notebooks, and Confluence doc instructions will be updated to let notebook submitters know to also submit a `.spec` test file along with their notebook.

TODO: Add `.spec` tests for all other notebooks in the directory.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by svetakvsundhar** [comment]
R: @damccorm 

Hi Danny,

Could you run the seed job on this newly created test, so I can test it out. Having some trouble testing it locally.

Also, if you can point me to an existing test that invokes the runner directly via gradle, that will be helpful. I looked in the repo and wasn't able to find any, so am not sure if I am invoking the runner correctly.

**Comment by damccorm** [comment]
Run Python_Notebooks PreCommit

**Comment by svetakvsundhar** [comment]
Run Python_Notebooks PreCommit

**Comment by damccorm** [comment]
A new seed job gets kicked off from master every 4 hours (or maybe 6 hours, I forget), so we need to rerun the seed job for this again before the `Run Python_Notebooks PreCommit` command would work.

I just did this and will kick off a job. In general, its better to iterate locally when possible though for new jobs

**Comment by svetakvsundhar** [comment]
Ah sorry I didn't have time to get to this today. Will work to land this once you're back.

**Comment by svetakvsundhar** [comment]
@damccorm , did some local iterations. Would it be possible to kick off another seedjob?

File: build.gradle.kts [hunk 1]
```diff
@@ -469,6 +469,16 @@ tasks.register("pythonPreCommit") {
...
 }
 
+tasks.register("pythonNotebooksPreCommit") {
+    /* doLast {
+      exec {
+        executable("sh")
+        args ("-c", ". ${envdir}/bin/activate && python -m apache_beam.examples.wordcount --runner DirectRunner --output /tmp/output.txt")
+      }
+     }*/
+  dependsOn(":sdks:python:wordCount")
```
Comment (damccorm) [inline build.gradle.kts:479]: This seems wrong - this should be depending on a task in `:examples:notebooks`, right?

**Comment by damccorm** [comment]
I kicked off https://ci-beam.apache.org/job/beam_SeedJob/11745/ and will run the job after it completes, I think this is wrong though as mentioned above

**Comment by damccorm** [comment]
Run Python_Notebooks PreCommit

**Comment by svetakvsundhar** [comment]
Run Python_Notebooks PreCommit

**Pull request closed without merging**